### PR TITLE
Docs: spelling and grammar fixes

### PR DESCRIPTION
**Pull request by scop**
----
📚 Documentation preview 📚: https://cpython-previews--122084.org.readthedocs.build/

### CODE REVIEW

File: Doc/library/fractions.rst [hunk 1]
```diff
@@ -92,7 +92,7 @@ another rational number, or from a string.
 
    .. versionchanged:: 3.9
       The :func:`math.gcd` function is now used to normalize the *numerator*
-      and *denominator*. :func:`math.gcd` always return a :class:`int` type.
+      and *denominator*. :func:`math.gcd` always return an :class:`int` type.
```
Comment (picnixz) [inline Doc/library/fractions.rst]:
```suggestion
      and *denominator*. :func:`math.gcd` always returns an :class:`int` type.
```

File: Doc/library/warnings.rst [hunk 2]
```diff
@@ -344,7 +344,7 @@ check::
 
 One can also cause all warnings to be exceptions by using ``error`` instead of
 ``always``. One thing to be aware of is that if a warning has already been
-raised because of a ``once``/``default`` rule, then no matter what filters are
+raised because of an ``once``/``default`` rule, then no matter what filters are
```
Comment (picnixz) [inline Doc/library/warnings.rst]: This one is correct.

Comment (freakboy3742) [reply Doc/library/warnings.rst]:
```suggestion
raised because of a ``once``/``default`` rule, then no matter what filters are
```

**Review by freakboy3742 — APPROVED** [review]
Thanks for the 👀 - lots of good fixes here.

**Pull request merged**